### PR TITLE
Alt prime helmet laurel

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
adds a normal golden laurel to the prime helmet alt styles like before.

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -947,6 +948,18 @@
 		usr.update_action_buttons()
 		return 1
 
+	if(src && choice && !M.incapacitated() && Adjacent(M))
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/helmet.dm:951]: If this is true then 939 is true and will never run.

Comment (Smokingsquirrel) [reply code/modules/clothing/head/helmet.dm:951]: So i just remove that as a whole?

Comment (Trilbyspaceclone) [reply code/modules/clothing/head/helmet.dm:951]: move the "if(choice == "prime laurel")" to where the first one is and have it be like "If( choice == laurel ||  choice ==whatever)"

Comment (Smokingsquirrel) [reply code/modules/clothing/head/helmet.dm]: Like that in my new commit?

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -932,12 +932,13 @@
...
 	options["prime saint"] = "prime_saint"
 	options["prime paladin"] = "prime_paladin"
+	options["prime laurel"] = "laurel_g"
 
 	var/choice = input(M,"What kind of style do you want?","Adjust Style") as null|anything in options
 
 	if(src && choice && !M.incapacitated() && Adjacent(M))
 		icon_state = options[choice]
-		if(choice == "prime saint")
+		if(choice == "prime saint" || "prime laurel")
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/helmet.dm:941]:
```suggestion
		if(choice == "prime saint" || choice ==  "prime laurel")
```

**Pull request merged**